### PR TITLE
Fixed CSS paths and HTML structure for GitHub Pages compatibility

### PR DESCRIPTION
**Pull request by TheHakan**
- Corrected file paths in `index.html` and `services.html` to ensure compatibility with GitHub Pages.
- Switched from absolute paths (e.g., `/style.css`) to relative paths (e.g., `./style.css`).
- Fixed minor HTML structure inconsistencies and improved consistency across files.

This resolves the issue where CSS and images did not load properly due to incorrect file paths.

### CODE REVIEW

**Comment by TheHakan** [comment]
@kahgrandini Please review this pull request. It resolves the file path issues for consistent styling and asset loading.

This pull request resolves the issue where CSS and images were not loading correctly on GitHub Pages due to incorrect file paths.
- Updated file paths in `index.html` and `services.html` to use relative paths (`./`) instead of absolute paths (`/`).
- Verified the changes locally and on GitHub Pages.

Please review and let me know if further changes are needed.

File: index.html [hunk 1]
```diff
@@ -1,86 +1,83 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Kart Racing Company - Rent Karts and Race Now!">
     <title>Kart Racing Company</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./style.css">
```
Comment (davorg) [inline index.html:9]: This change makes no difference at all. `style.css` and `./style.css` both load the file from the current directory.

Comment (TheHakan) [reply index.html:9]: You are right @davorg! Both paths work similarly in most environments, this change ensures compatibility with GitHub Pages, where absolute paths often cause issues due to how the environment serves files.

Comment (davorg) [reply index.html:9]: > Both paths work similarly in most environments

No. Both paths work exactly the same in all environments. If an environment works differently, it is breaking the HTTP specification.

Comment (kahgrandini) [reply index.html:9]: Thank you so much @TheHakan @davorg ! It resolved the problem.

**Pull request merged**